sql/flow.go

```diff
@@ -73,17 +73,18 @@ func displayMessages(r io.Reader) error {
 	return nil
 }
 
-func CommonDockerUtil(cmd, args []string, flags map[string]string, mountDirs []string) error {
+func CommonDockerUtil(cmd, args []string, flags map[string]string, mountDirs []string) (int, error) {
+	var statusCode int
 	ctx := context.Background()
 
 	cli, err := DockerClientInit()
 	if err != nil {
-		return fmt.Errorf("docker client initialization failed %w", err)
+		return statusCode, fmt.Errorf("docker client initialization failed %w", err)
```

Comment:
where is the value of statusCode set?

Reply:
The default int is 0 :)

Reply:
@feluelle, what if we set it explicitly, to help the readability? Does lint complain?

Reply:
Yes, it complains.
```
sql/flow.go:78:23: var-declaration: should drop = 0 from declaration of var statusCode; it is the zero value (revive)
        var statusCode int = 0
                             ^
make: *** [lint] Error 1
```